### PR TITLE
Supress Terraform warnings if terraformrc file is not present

### PR DESCRIPTION
**Pull request by tibeer**
Signed-off-by: Tim Beermann <beermann@osism.tech>

### CODE REVIEW

File: terraform/Makefile [hunk 1]
```diff
@@ -28,7 +28,9 @@ endif
 export OS_CLOUD ?= $(ENVIRONMENT)
 
 export TF_VAR_cloud_provider=$(ENVIRONMENT)
-export TF_CLI_CONFIG_FILE=terraformrc
+ifneq (,$(wildcard terraformrc))
+  export TF_CLI_CONFIG_FILE ?= terraformrc
```
Comment (osfrickler) [inline terraform/Makefile:32]: Do we need this at all? Maybe we can rather just drop setting this variable?

Comment (tibeer) [reply terraform/Makefile:32]: We also have [this](https://github.com/osism/testbed/blob/main/terraform/terraformrc.sample) file. Should be dropped than as well. I have no strong opinion on this topic. @berendt what do you think?

Comment (berendt) [reply terraform/Makefile:32]: I would continue to support the file itself and not remove it completely. Sometimes you need the terraformrc to customize things.

Comment (tibeer) [reply terraform/Makefile:32]: let's keep it then.

**Pull request merged**